tracing-futures/src/lib.rs

```diff
@@ -105,10 +105,8 @@
 #[cfg(feature = "std-future")]
 use pin_project::pin_project;
 
-pub(crate) mod stdlib;
-
 #[cfg(feature = "std-future")]
```

Comment:
we can probably just remove the feature flag for `std::future` in the upcoming release.... it was added _way_ back in the day before `std::future` was stable, so it could be enabled by users on nightly. now, it's stable in our MSRV, so this is definitely no longer necessary.

Reply:
Do you want me to remove all the uses of the feature here? I can keep `std-future = []` as a no-op so it doesn't break existing code.

Reply:
Oh actually it looks like enabling `std-future` conflicts with `futures_01` ... do you plan to remove `futures_01` in tracing 0.2?